### DISCUSSION TITLE
Have Chatterino remember what audio source it's set to in sound settings

### DISCUSSION THREAD

**Discussion by Zuelatak** [Chatterino/chatterino2#3240] [Ideas]
Whenever I open Chatterino it defaults the audio to my default device instead of the one I have set in Sound Settings. I have to change the device and then change it back in order for it to send to the proper channel. Would love it if Chatterino remembered where it's supposed to go or if an option could be added to the settings to tell it where to go and have it remember that.

**Comment by ALazyMeme** [comment]
You're not being clear here. Do you use virtual audio cables or something? It is completely normal for Chatterino to play through the default audio device in Windows.

**Comment by Zuelatak** [comment]
Virtual audio cables and physical cables have this problem. 
![image](https://user-images.githubusercontent.com/41973452/233733726-ac806c64-f24d-428f-ab60-e3e93bdf86ac.png)

The output is being told to go to my TV in this example, but still goes to the default device unless I switch it off the TV and then back onto the TV.

**Comment by Infinitay** [comment]
Could try using EarTrump and set Chatterino to only play through which ever device you want and see if you still have the same issue

**Comment by Coldblackice** [comment]
I've also recently been having sound issues with Chatterino (Chatterino Nightly 2.3.4 (commit b0e79b603)) where sounds don't play anymore despite other programs still able to. I use EarTrumpet and it's not an issue of Chatterino's OS mixer volume control.

I haven't pinpointed yet where it's stemming from, but nothing has changed audio device / OS update wise. There might be an issue with highlights that overlap with each other that have separate sound alerts, perhaps a clashing of priority. Just putting this out there wondering if anyone else is seeing this.
